<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FACTab</class>
 <widget class="QWidget" name="FACTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="1" rowspan="2">
    <widget class="QToolButton" name="recordButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #960000;
color: white;</string>
     </property>
     <property name="text">
      <string>&amp;Record</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Buffer size</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="10">
    <widget class="Waveform" name="facWaveform">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QComboBox" name="facSizeCombo"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="unitCheck">
     <property name="text">
      <string>Delay in samples</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCheckBox" name="detectPeaksCheck">
     <property name="text">
      <string>Detect peaks:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Averaging:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QSlider" name="averagingSlider">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>70</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="6" colspan="2">
    <widget class="ContextAwareSpinBox" name="sigmaSpin">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> Ïƒ</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
   <customwidget>
     <class>ContextAwareSpinBox</class>
     <extends>QDouble</extends>
     <header>ContextAwareSpinBox.h</header>
   </customwidget>
  <customwidget>
   <class>Waveform</class>
   <extends>QFrame</extends>
   <header>Waveform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
